### ISSUE TITLE
Wepage not showing, console + log crashing

### ISSUE THREAD

**Issue by Rene1695** [ich777/docker-magicmirror2#3]
Hi,

I think this is related to #1 and #2. I just updated my magic mirror to 2.25.0 and now I can not access my webpage anymore. I disabled my pinhole completely but still the same issue.

I completly reinstalled the image and put It in a clean new directory (magicmirror4). I just get this to "could not get version number, putting server into sleep mode". If I open the console there is some glitching? (please see my video on this). I can not type something into the console and there are different error messages. I could not catch this up as text, I think this only can be seen in the video. 

If I use my standard config (magicmirror3) which was running for a year, I got the same glitching in the console and this in the log file: 

![image](https://github.com/ich777/docker-magicmirror2/assets/71972934/c630a6fc-faa0-47c9-9f88-8fb6306a8889)

Any ideas on this? I think it's very similar to #1 and #2 but the glitching the console is weird. Everything else in unraid seems to work fine. 

Video link: 
https://www.dropbox.com/scl/fi/oyx9vf1p954oyi9wu6dkr/MagicMirror_issue.mov?rlkey=anfozsbcrd8jrtno8l5jczwi5&dl=0

**Comment by ich777** [comment]
Sorry but I really can't reproduce that at all...

How often did you try to start the container? Please not that I use the GitHub API to determine the version number and you only have about 50 requests per one hour and some users run into issues if the container is looping because each start/restart from the container is one request from the API.

I've upload the log from a completely fresh installed container: [magicmirror_log.txt](https://github.com/ich777/docker-magicmirror2/files/12910410/magicmirror_log.txt)

And also a screenshot from the UI itself:
![grafik](https://github.com/ich777/docker-magicmirror2/assets/28066518/8f549baa-bc26-41a2-bd70-5afec526ee31)

BTW, you are on Unraid 6.11.5 which is not recommended because the runc version and the glibc version are not compatible and this could be the reason for this behavior, here is also a screenshot from the container console:
![grafik](https://github.com/ich777/docker-magicmirror2/assets/28066518/553a88a3-11e0-4ee6-9f5e-6e9d8889256c)



I can't reproduce that over here and my assumption is that you really should do a update from the base OS, you are behind a few versions.
I also have to say that I won't give further support until you update Unraid since as said, you are a few versions behind and this is most likely the case why it is not working.

**Comment by Rene1695** [comment]
Hi thank you very much for the quick response!

I think it's because I am on the stable branch which is up to date. I tested switching to the "next" branch and installed unraid 6.12.4 but after installation I saw a lot of changes within the OS. Also no docker container was working, I could not start any container and on edit I saw that there was no network config set. I have set all docker to a custom IP address and that seems to not work anymore?

To be honest I am a little bit scared of using this new version, there seems to be a lot of time needed until everything else is working again. I have quite a lot of smart home stuff running and this would be good to work :D do you have any tips when upgrading to this new version?

**Comment by Rene1695** [comment]
So I checked multiple times but the option "Docker custom network type" in the "Docker" --> "Settings" tab has been removed. In 6.11.5 I set this to "macvlan".

I have set static IP adress to all my dockers with the custom br0 network type. Unfortunately I can not enable it again so I would need to switch all my dockers into bridge mode (which will crash at different places because it set IPs in different config files).

Do you know a work-around?

By the way: I set the magic mirror docker into bridge mode and started the container within 6.12.4 but I got more or less the same behavior. The Log and console window will crash after some seconds and the webpage of magic mirror is not showing.

**Comment by ich777** [comment]
> To be honest I am a little bit scared of using this new version, there seems to be a lot of time needed until everything else is working again. I have quite a lot of smart home stuff running and this would be good to work :D do you have any tips when upgrading to this new version?

You should really read the release notes for the 6.14 releases because there is a MACVLAN fix for Fritzbox users in it which only requires you to change one setting if you where using static IP addresses for container (which I not recommend because it's most of the time not necessary).

But staying on a outdated version is for me even more scary and makes no sense at all, you are missing many security updates with 6.11.5 and this is nothing to laugh about TBH...

You should really upgrade your system, you can't stay forever on 6.11.5
Just read the release notes and create a Bug report if something is not working, there are many users out there and they all have no issues with 6.12.4

Again, I can't help with such an outdated version (Unraid 6.11.5 was released: 2022-11-21 so to speak almost a year ago!)

**Comment by ich777** [comment]
> So I checked multiple times but the option "Docker custom network type" in the "Docker" --> "Settings" tab has been removed. In 6.11.5 I set this to "macvlan".

This is just normal if you don't have enabled the bridge and it will indeed run as macvlan network. The main reason why Unraid crashed was because most people had enabled the bridge on the interface while using macvlan, this ultimately causes crashes because you can't use a bridged interface with a macvlan network.


> I have set static IP adress to all my dockers with the custom br0 network type. Unfortunately I can not enable it again so I would need to switch all my dockers into bridge mode (which will crash at different places because it set IPs in different config files).

Then make a change in the Docker template to use now the network eth0 instead of br0 and everything will be back to a working state as usual.
If that doesn't work, make a change to the template, revert the change back and click Apply (you will need to make a change because otherwise you can't click the Apply button.



> By the way: I set the magic mirror docker into bridge mode and started the container within 6.12.4 but I got more or less the same behavior. The Log and console window will crash after some seconds and the webpage of magic mirror is not showing.

What means more or less? Is it the same or not, as said, I can't reproduce this over here.
Do you maybe have some kind of AdBlocking (like PiHole, AdGuard,...) or some special Firewall rules in place that block access to certain domains (Unifi routers are known to cause issues).

**Comment by ich777** [comment]
@Rene1695 any update on this? I tried it again and everything is working for me, another member from the forums also tried it and it is working just fine.

**Comment by Rene1695** [comment]
@ich777 I finally got 6.12.4 running and also my custom br0 is now working again. After a clean re-install of magic mirror I can confirm that this is now working. 

Sadly, if I set the fresh install to my "old" magic mirror config (where I installed all my modules in 6.11.5), than magic mirror is not starting and I have the same issue. 

So I think I need to install all the modules in the new instance and move my config.js from my old config to my new config (I hope this is working). I noticed that the structure has changed within magic mirror. Normally the modules were installed in the "magic mirror/modules" folder but I don't have this folder anymore. Only the "modules.txt". Do I need to list all modules here or how do I install third party modules now?

Thank you for all the help!!
